src/core/core.scale.js

```diff
@@ -80,7 +80,11 @@ function getLineValue(scale, index, offsetGridLines) {
 	var lineValue = scale.getPixelForTick(index);
 
 	if (offsetGridLines) {
-		if (index === 0) {
+		if (scale.getTicks().length === 1) {
```

Comment:
I would leave this comment on the function name, but github won't let me since it wasn't changed. What do you think about renaming this function to something like `getPixelForGridLine`?

Reply:
Agreed, `getPixelForGridLine` looks better. I will rename it.

Reply:
I also agree, but `getLineValue` is public and that would be a breaking change. Also, introducing a new method would need to be private and thus `_` prefixed: `_getPixelForGridLine`

Reply:
@simonbrunel are you sure it's public? I thought each `function` at the top of the file was private and only the methods inside `module.exports` were public

Reply:
You right, I have been mistaken by the extra indentation and thought it was a public method. Please ignore my previous comment and feel free to rename.

Reply:
Sorry, just thinking about this a little bit more, I wonder if `getPixelForGridLine` is really the right name. It's such a small method and only called from one place, I think actually the clearest thing to do would be to inline the function. That makes it easier to follow the code because we don't have to navigate between the top and bottom of the file to see what the flow of the code is. What do you think about that?

Reply:
Sorry, I got mixed up here. Nevermind